packages/webpack-stats-report/src/createReport.ts

```diff
@@ -91,15 +91,19 @@ export function createDetailedReport(
   const comparisonLink = reportData.comparisonToolUrl
     ? `<a target="_blank" rel="noopener noreferrer" href="${reportData.comparisonToolUrl}">🔍</a>`
     : "";
+    const percentageChange = Math.floor((100 / reportData.totalSize) * Math.abs(reportData.totalDiff));
   const deltaSizeMessage = `${getReducedOrIncreased(
-    reportData.totalDiff,
-    minimumIncrease
-  )} total size by ${formatBytes(Math.abs(reportData.totalDiff))}`;
-  const totalSizeMessage = `Total size: ${formatBytes(reportData.totalSize)}`;
+    reportData.totalDiff
+  )} by <b>${percentageChange}%</b>(${formatBytes(Math.abs(reportData.totalDiff))})`;
+  const totalSizeMessage = `${formatBytes(reportData.totalSize)}`;
   const ownersMessage = reportData.ownedBy?.map((owner) => `@${owner}`).join(" ") || "";
-  const prefix = `<summary><span style="font-size: 16px">${reportData.name} ${comparisonLink}</span><br><ul><li>${deltaSizeMessage}</li><li>${totalSizeMessage}</li>${shouldAtMention(reportData,atMentionThreshold) ? `<li>${ownersMessage}</li>` : ``}</summary>`;
 
-  const header = "\n| Asset name | Size | Diff | |";
+  const prefix = `<summary><span style="font-size: 16px">  ${comparisonLink} ${reportData.name} ${deltaSizeMessage} to ${totalSizeMessage} ${shouldAtMention(reportData,atMentionThreshold) ? `${ownersMessage}` : ``}</span> </summary>`;
+  
+  
+  const shouldTableBeExpanded = percentageChange >= 20;
```

Comment:
Sorry if we forgot to update this particular ticket, but we decided to start with 5% rather than 20%

Reply:
Yeah. But that's alright. Should be good now @p01